test/unit/specs/components/staking/TableValidators.spec.js

```diff
@@ -59,17 +59,15 @@ describe(`TableValidators`, () => {
 
   it(`should create an enriched validator object for a signed in user`, () => {
     expect(wrapper.vm.enrichedValidators[0].small_moniker).toBe(`mr_mounty`)
-    expect(wrapper.vm.enrichedValidators[0].percent_of_vote).toBe(
-      0.27999944000112
-    )
+    expect(wrapper.vm.enrichedValidators[0].voting_power).toBe(`0.0000279999`)
```

Comment:
seems very small (and no percent sign?)

Reply:
This is the fraction (14 tokens of 500001 total bonded tokens). I changed it to be more speaking.